js/welcome.js

```diff
@@ -0,0 +1,19 @@
+const splash_overlay = document.getElementById("splash-overlay");
+const welcome_splash_1 = document.getElementById("welcome-splash-1");
+const welcome_splash_2 = document.getElementById("welcome-splash-2");
```

Comment:
(nit) names like `welcome_message` and `welcome_fancy_menu` would be clearer for me to distinguish sections